lib/l10n/de_DE/app_de_DE.arb

```diff
@@ -645,6 +645,8 @@
   "@scanIntoLocation": {},
   "scanIntoLocationDetail": "Artikel per Scan zu Lagerort hinzufügen",
   "@scanIntoLocationDetail": {},
+  "scanReceivedParts": "Empfangene Artikel scannen",
```

Comment:
Translations are handled over on [crowdin](https://crowdin.com/project/inventree) - don't directly add translations to the source here, might cause conflicts

Reply:
Ah, didn't know that, removed it!